### PR TITLE
Update Chinese translation

### PR DESCRIPTION
**Pull request by littlebear0729**

### CODE REVIEW

**Comment by HB9HIL** [comment]
Hello

Yesterday there was another language PR merged.

Please pull the latest dev in your branch and solve the conflicts.

Thank you for your work!

**Comment by littlebear0729** [comment]
Sorry for the inconvenience. I missed the dev branch and just did the work on master branch.
Now I've merge the dev branch. Should I re-open a pull-request to the dev branch or this pr is just fine?

**Comment by HB9HIL** [comment]
No change it to dev branch please. We only merge PR's to dev branch

**Comment by littlebear0729** [comment]
Done. Cloudlog is really good to use, hope this project will be better!

This is Haonan(BI1NEI) from Beijing, China. 73!

**Comment by HB9HIL** [comment]
> Done. Cloudlog is really good to use, hope this project will be better!
> 
> This is Haonan(BI1NEI) from Beijing, China. 73!

Great work 👍 

In `dev` branch there are much more language snippets to translate into chinese. Can you do that aswell? 😄

**Comment by littlebear0729** [comment]
Of course! I will do it when I have more spare times.

**Comment by HB9HIL** [comment]
> Of course! I will do it when I have more spare times.

Thank you very much! We continuously adding more multilanguage support to Cloudlog. 👍 We are working with `dev` branch 😄

File: application/language/chinese_simplified/general_words_lang.php [hunk 1]
```diff
@@ -117,8 +117,7 @@
 $lang['gen_hamradio_prefix'] = "Prefix";
 $lang['gen_hamradio_suffix'] = "Suffix";
 $lang['gen_hamradio_mode'] = '模式';
-$lang['gen_hamradio_dx'] = 'Dx';
-$lang['gen_hamradio_mode'] = 'Mode';
+$lang['gen_hamradio_dx'] = 'DX';
```
Comment (HB9HIL) [inline application/language/chinese_simplified/general_words_lang.php:120]: Please check that. You removed "$lang['gen_hamradio_mode'] = 'Mode';"

Comment (HB9HIL) [reply application/language/chinese_simplified/general_words_lang.php:120]: Did you pull the latest `dev` branch?

Comment (littlebear0729) [reply application/language/chinese_simplified/general_words_lang.php:120]: Yes, line 119 and line 121 of the original file is duplicated with same key 'gen_hamradio_mode', this cause the string translation is overwritten.

Comment (HB9HIL) [reply application/language/chinese_simplified/general_words_lang.php:120]: ahhh omg haha here in switzerland it's 8:42 am sorry my bad

**Review by HB9HIL — APPROVED** [review]
approved :)

**Pull request merged**